published/AWS/servers_by_account_and_tags/function.js

```diff
@@ -0,0 +1,119 @@
+function(account_label, env_tag_value, ellipsis) {
+  "use strict"; 
+const Q = require('q');
+var AWS = require('aws-sdk');
+var _ = require('underscore');
+
+// -----------------------------------------------------------------------------------------------
+// INPUTS
+// -----------------------------------------------------------------------------------------------
+
+// Add here the region codes of the regions you want Ellipisis to search for Instances
+var regions = [
+  'us-east-1',
+  'us-west-1'
+]
+
+// What tags do you want to use to select the instances?
+// Add the Tag Key and the Tag Value. Tag Value can be an input param.
+const searchedTags = [
+  {Key: "Environment", Value: env_tag_value}
+];
+
+// What aws accounts do you want to use?
+// Add here the accounts you want Ellipisis to use. The label is used to 
+// allow the user to choose what account to use.
+var accounts = [
+  {
+    label: "mb",
+    key: ellipsis.env.MB_AWS_KEY,
+    secret: ellipsis.env.MB_AWS_SECRET
+  },
+  {
+    label: "ep",
+    key: ellipsis.env.EP_AWS_KEY,
+    secret: ellipsis.env.EP_AWS_SECRET
+  }
+];
+
+// -----------------------------------------------------------------------------------------------
+// Given AWS account and a list of regions it finds all the 
+// instances in the regions that match a given list of tags.
+// tags are object with a Key and Value properties. For Example:
```

Comment:
"are objects with Key and Value properties"

Also "For example:" — no capital E necessary.